seed/challenges/02-data-visualization-certification/react-projects.json

```diff
@@ -202,6 +265,25 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте Рогулайк Данжеон-Кроулер",
```

Comment:
Dungeon Crawler does not have to be Roguelike. It also has a particular translation.
Roguelike - https://ru.wikipedia.org/wiki/Roguelike
Dungeon Crawler - https://ru.wikipedia.org/wiki/%D0%9F%D0%BE%D0%B4%D0%B7%D0%B5%D0%BC%D0%B5%D0%BB%D1%8C%D0%B5_%28%D1%80%D0%BE%D0%BB%D0%B5%D0%B2%D1%8B%D0%B5_%D0%B8%D0%B3%D1%80%D1%8B%29

so it should be, for example, `Создайте Рогулайк-игру Подземелье` or `Создайте Рогалик-игру Подземелье`, latter does not fit, mainly because there is an actual word `рогалик` with different meaning.

Are you sure, `Создайте игру Dungeon Crawler` should be used?